README.md

```diff
@@ -1,20 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# React-Redux setup
+# Covid Metrics
 
-This is a simple yet powerful React-Redux boilerplate for later use as a starting point in other projects. It derives from `create-react-app` with a few modifications, which include installing StyleLint, ESLint, and Bootstrap, removing unimportant files, and adding some content to the remaining files. It features a functional Counter component that implements Redux for state management, using `useSelector` and `useDispatch` hooks.
-
-## Mobile
+This is a mobile web application that features 2 pages: The homepage and the details page. The homepage displays all today's statistics about confirmed cases, deaths, and recoveries caused by Covid-19 in the world, as a total and separated by countries. When the user clicks on any country, the website navigates to the details page, which contains the detailed statistics about that specific country as a total and separated by regions. Both pages contain a search bar that allows the user to filter by country name. The data is consumed from [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). Built with React, Redux, and Bootstrap.
```

Comment:
- **[OPTIONAL]** For better UX , it would be nice if the cursor type on hovering over your country cards were `pointer` so that users on desktop know that they are clickable.